pom.xml

```diff
@@ -3,7 +3,7 @@
   <parent>
     <groupId>org.jenkins-ci.plugins</groupId>
     <artifactId>plugin</artifactId>
-    <version>1.400</version>
+    <version>1.620</version>
```

Comment:
I't better to keep the plugin compatible with recent LTS versions if there is no strong need to bump the depencency